pkg/koko/task.go

```diff
@@ -115,6 +117,8 @@ func uploadRemainFTPFile(jmsService *service.JMService) {
 		}
 		return nil
 	})
+	logger.Infof("Start upload remain %d ftp files 10 min later ", len(allRemainFiles))
+	time.Sleep(10 * time.Minute)
 
 	for absPath, remainFTPFile := range allRemainFiles {
 		absGzPath := absPath
```

Comment:
There aren't any immediate irregularities or potential issues with the code you've provided. However, there might be a performance improvement if both functions were combined and called only once when needed (e.g., one before initiating FTP uploads). For optimization suggestion:

1. Use strings instead of variables where possible to avoid unnecessary memory allocation.

For example: Replace `allRemainFiles` array usage with string indexing like below.

```go
for absPath, remainFTPFile := range allRemainFiles {
     ...
}
```
This is more efficient than storing an entire slice in memory at once because each index will now point directly to individual filenames rather than having to fetch all elements from the slice first.

The updated version would look something like this:
```go
time.Sleep(10 * time.Minute)

for fileIndex := range allRemainFiles {  // use index variable here
   logger.Infof("Start upload remain %d ftp files 20 min later ", len(allRemainFiles))
   time.Sleep(20 * time.Minute)  // change to 20 minutes as specified

   // Assume these parts don't take many resources so they can be removed
   ...

// continue processing other operations...
}
```